resources/js/Pages/Thread/Show.jsx

```
@@ -0,0 +1,53 @@
+import { Head } from '@inertiajs/react'
+import { SideMenu } from '../../Components/SideMenu'
+import { LogoutButton } from '../../Components/LogoutButton'
+import { HiMicrophone, HiSpeakerphone } from 'react-icons/hi'
+
+export default function Show({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)